ansible_collections/arista/avd/roles/eos_designs/templates/network_services/vlan-interfaces.j2

```diff
@@ -75,6 +75,25 @@ vlan_interfaces:
         vrf: {{ svi_ip_helpers[helper_ip].source_vrf | arista.avd.default }}
 {%                 endfor %}
 {%             endif %}
+{%             if svi.ospf.enabled is arista.avd.defined(true) and vrf.ospf.enabled is arista.avd.defined(true) %}
```

Comment:
I think you want this `if` as a condition for all the OSPF knobs below. Not only area and point-to-point.

Reply:
Yep, thanks, applied this to vlan-interfaces and ethernet-interfaces as well.